boards/arm/efr32mg_sltb004a/doc/efr32mg_sltb004a.rst

```diff
@@ -0,0 +1,202 @@
+.. _efr32mg_sltb004a:
+
+EFR32MG-SLTB004A
+###############
+
+Overview
+********
+
+The EFR32™ Mighty Gecko Starter Kit EFR32MG-SLTB004A (a.k.a Thunderboard
+Sense 2) contains a MCU from the EFR32MG family built on ARM® Cortex®-M4F
+processor with low power capabilities.
+
+.. image:: efr32mg_sltb004a.jpg
+   :width: 640px
+   :align: center
+   :alt: EFR32MG-SLTB004A
+
+Hardware
+********
+
+- EFR32MG12 Mighty Gecko Wireless SoC with 38.4 MHz operating frequency
+- ARM® Cortex® M4 core with 256 kB RAM and 1024 kB Flash
+- Macronix ultra low power 8-Mbit SPI flash (MX25R8035F)
+- 2.4 GHz ceramic antenna for wireless transmission
+- Silicon Labs Si7021 relative humidity and temperature sensor
+- Silicon Labs Si1133 UV index and ambient light sensor
+- Silicon Labs Si7210 hall effect sensor
+- Bosch Sensortec BMP280 barometric pressure sensor
+- ams CCS811 indoor air quality gas sensor
+- TDK InvenSense ICM-20648 6-axis inertial sensor
+- TDK InvenSense ICS-43434 MEMS microphone
+- Four high brightness RGB LEDs from Broadcom Limited (ASMT-YTB7-0AA02)
+- One bi-color LED and two push buttons
+- Power enable signals for fine grained power-control
+- On-board SEGGER J-Link debugger for easy programming and debugging, which
+includes a USB virtual COM port
```

Comment:
this line should be indented to the ``O`` on the previous line

Reply:
Addressed.